src/AES.java

```diff
@@ -0,0 +1,50 @@
+package src;
+
+import java.util.Base64;
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.IvParameterSpec;
+
+// This is a child of Encryption class 
+
+public class AES extends Encryption {
+
+    // Overwrites the Encrypt class to implement the encryption of AES
+
+    static byte[] cipherText;
+    static byte[] plainText;
+
+    public static String encrypt(String algorithm, String input, SecretKey key,
```

Comment:
Be sure all public classes and methods have meaningful Javadoc.  Because you are overriding parent's encrypt method this method can't be static